PubSub.Application/Models/CreateSubscriptionParameters.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace PubSub.Application.Models
+{
+    public class CreateSubscriptionParameters
+    {
+        public CreateSubscriptionParameters(string callback, string topic)
        {

```

Comment:
There's an issue with the line ending in this file.

I see a ^M at the end of this line, and VS shows a warning about the endings